src/bitcoin-wallet.cpp

```diff
@@ -31,6 +31,7 @@ static void SetupWalletToolArgs(ArgsManager& argsman)
     argsman.AddArg("-dumpfile=<file name>", "When used with 'dump', writes out the records to this file. When used with 'createfromdump', loads the records into a new wallet.", ArgsManager::ALLOW_STRING, OptionsCategory::OPTIONS);
     argsman.AddArg("-debug=<category>", "Output debugging information (default: 0).", ArgsManager::ALLOW_ANY, OptionsCategory::DEBUG_TEST);
     argsman.AddArg("-descriptors", "Create descriptors wallet. Only for 'create'", ArgsManager::ALLOW_BOOL, OptionsCategory::OPTIONS);
+    argsman.AddArg("-legacy", "Create legacy wallet. Only for 'create'", ArgsManager::ALLOW_BOOL, OptionsCategory::OPTIONS);
```

Comment:
23002: @knst please evaluate if we are ready for descriptor by default. I thought we had more we needed to fix / implement

Reply:
There is still missing mnemonic feature for descriptor wallets, which is a blocker IMO

I'll work on it for 22.1+

Reply:
dropped 23002

Reply:
should drop 24281